homeassistant/components/surepetcare/__init__.py

```diff
@@ -45,101 +32,62 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+PLATFORMS = ["binary_sensor", "sensor"]
+SCAN_INTERVAL = timedelta(minutes=3)
 
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_USERNAME): cv.string,
-                vol.Required(CONF_PASSWORD): cv.string,
-                vol.Optional(CONF_FEEDERS, default=[]): vol.All(
-                    cv.ensure_list, [cv.positive_int]
-                ),
-                vol.Optional(CONF_FLAPS, default=[]): vol.All(
-                    cv.ensure_list, [cv.positive_int]
-                ),
-                vol.Optional(CONF_PETS): vol.All(cv.ensure_list, [cv.positive_int]),
-                vol.Optional(
-                    CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
-                ): cv.time_period,
-            }
+            vol.All(
+                {
+                    vol.Required(CONF_USERNAME): cv.string,
+                    vol.Required(CONF_PASSWORD): cv.string,
+                    vol.Optional(CONF_FEEDERS): vol.All(
+                        cv.ensure_list, [cv.positive_int]
+                    ),
+                    vol.Optional(CONF_FLAPS): vol.All(
+                        cv.ensure_list, [cv.positive_int]
+                    ),
+                    vol.Optional(CONF_PETS): vol.All(cv.ensure_list, [cv.positive_int]),
+                    vol.Optional(CONF_SCAN_INTERVAL): cv.time_period,
+                },
+                cv.deprecated(CONF_FEEDERS),
+                cv.deprecated(CONF_FLAPS),
+                cv.deprecated(CONF_PETS),
+                cv.deprecated(CONF_SCAN_INTERVAL),
+            )
         )
     },
     extra=vol.ALLOW_EXTRA,
 )
 
 
-async def async_setup(hass, config) -> bool:
-    """Initialize the Sure Petcare component."""
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
+    """Set up the openSenseMap air quality platform."""
```

Comment:
It's still saying it's setting up the openSenseMap air quality platform. 😄

Reply:
```suggestion
    """Set up the Sure Petcare integration."""
```